daemon/daemon_unix.go

```diff
@@ -419,6 +419,7 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 		netlabel.DriverMTU:        strconv.Itoa(config.Mtu),
 		bridge.EnableIPMasquerade: strconv.FormatBool(config.Bridge.EnableIPMasq),
 		bridge.EnableICC:          strconv.FormatBool(config.Bridge.InterContainerCommunication),
+		netlabel.EnableIPv6:       strconv.FormatBool(config.Bridge.EnableIPv6),
```

Comment:
To elaborate more:

```
root@lp01679:/usr/bin# docker network create -o com.docker.network.enable_ipv6=true newbridge
13fc7ada2827d349d21a2c32fdd5a6fec32b6c0b8f591b7ec80d1dc4156146e1
root@lp01679:/usr/bin# docker network inspect newbridge
        "options": {
            "com.docker.network.enable_ipv6": "true"
        }
    }
]
root@lp01679:/usr/bin# ip addr
[...]
394: br-13fc7ada2827: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN 
    link/ether 02:42:8b:20:12:de brd ff:ff:ff:ff:ff:ff
    inet 172.18.0.1/16 scope global br-13fc7ada2827
       valid_lft forever preferred_lft forever
    inet6 fe80::1/64 scope link tentative 
       valid_lft forever preferred_lft forever
root@lp01679:/usr/bin# docker network rm newbridge
root@lp01679:/usr/bin# docker network create -o com.docker.network.enable_ipv6=false newbridge
410ec2597a8ed01125fc113ab1c51c37e55d481df2d82a50bd731b95702cfa0f
root@lp01679:/usr/bin# docker network inspect newbridge
[...]
        "options": {
            "com.docker.network.enable_ipv6": "false"
        }
    }
]
root@lp01679:/usr/bin# ip addr
[...]
395: br-410ec2597a8e: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN 
    link/ether 02:42:71:ff:88:ed brd ff:ff:ff:ff:ff:ff
    inet 172.18.0.1/16 scope global br-410ec2597a8e
       valid_lft forever preferred_lft forever
```

i.e. you can detect ipv6 on a network by looking at labels. It doesn't make sense to me that the default bridge would be a special case.

In fact, this is the only place in Docker where `NetworkOptionGeneric` is used! Although some of the labels might partition into driver-specific options, network specific options and generic options, that's a decision for the libnetwork internals to make.

Of course, I've only been playing around libnetwork for a day, so I may be misunderstanding.